service/service.go

```diff
@@ -34,6 +35,13 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 
 	// ADD CODE: Add other(s) to serviceList here
```

Comment:
Consider removing this comment now that services are being initialised.

Reply:
As we are going to add other client to service list, I thought it might be better to keep the comments